src/components/MaterialUI/Inputs/DatePicker/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import styled, { createGlobalStyle } from "styled-components";
+import { getThemeProp } from "../../../../utils";
+import Icon from "../../../../components/Icon";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import TimePicker from "./TimePicker";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => ({
+	datePickerWrapper: {
+		display: "flex",
+		flexWrap: "wrap",
+		width: "auto",
+		paddingLeft: "5px",
+	},
+}));
+
+const CalendarIcon = styled(Icon).attrs(props => ({
+	id: getThemeProp(["icons", "date"], "calendar")(props),
+}))`
+	font-size: 20px;
```

Comment:
font size shouldn't b hardcoded and should be taken from theme. If there isn't font size 20px in theme then it should be defined (ask Alain how to name it).